src/plugins/search/search.js

```diff
@@ -16,7 +16,9 @@ function escapeHtml(string) {
 function getAllPaths(router) {
   const paths = []
 
-  document.querySelectorAll('.sidebar-nav a:not(.section-link):not([data-nosearch])').forEach(node => {
+  const nodes = document.querySelectorAll('.sidebar-nav a:not(.section-link):not([data-nosearch])')
```

Comment:
We can use `Docsify.dom.findAll` see [source code](https://github.com/docsifyjs/docsify/blob/a5f333a329aad087588b7fd60dae6d867f24f99a/src/core/util/dom.js#L45)